hetmatpy/testing.py

```diff
@@ -2,7 +2,7 @@
 
 import hetio.readwrite
 
-import hetmech.hetmat
```

Comment:
Why do we want `hetmatpy/testing.py` outside the `tests` directory? Could test files not do `import .testing` if `tests` is not a module with an `__init__.py` file?

Reply:
Edit: This comment should have been on `hetmatpy/testing.py`

Reply:
From commit message in https://github.com/hetio/hetmatpy/pull/2/commits/8ee8b3707fef3bdfe3495ed0c1132645fc95b05c:

Attempt to fix pytest error https://travis-ci.com/hetio/hetmatpy/jobs/159147253#L487

```
____________ ERROR collecting hetmatpy/tests/test_degree_weight.py _____________
import file mismatch:
imported module 'hetmatpy.tests.test_degree_weight' has this __file__ attribute:
  /home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/hetmatpy/tests/test_degree_weight.py
which is not the same as the test file we want to collect:
  /home/travis/build/hetio/hetmatpy/hetmatpy/tests/test_degree_weight.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

Basically, there was an annoying pytest error where having __init__ in a `tests` sub-directory was causing an error. However, removing it prevents importing across files.